--- a/EditorComponents/EditorTerrainSystemComponent.cpp
+++ b/EditorComponents/EditorTerrainSystemComponent.cpp
@@ -8,14 +8,11 @@
 
 #include <AzCore/Serialization/SerializeContext.h>
 #include <EditorComponents/EditorTerrainSystemComponent.h>
-#include <TerrainRenderer/EditorComponents/EditorTerrainMacroMaterialComponent.h>
 
 namespace Terrain
 {
     void EditorTerrainSystemComponent::Reflect(AZ::ReflectContext* context)
     {
-        GradientSignal::ImageCreatorUtils::
-            PaintableImageAssetHelper<EditorTerrainMacroMaterialComponent, EditorTerrainMacroMaterialComponentMode>::Reflect(context);
 
         if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
