--- a/TerrainRenderer/TerrainDetailMaterialManager.h
+++ b/TerrainRenderer/TerrainDetailMaterialManager.h
@@ -58,6 +58,11 @@
         void Initialize(
             const AZ::Data::Instance<AZ::RPI::ShaderResourceGroup>& terrainSrg,
             const AZ::Data::Instance<AZ::RPI::Material>& materialInstance);
+        void SetTerrainMaterial(const AZ::Data::Instance<AZ::RPI::Material>& material)
+        {
+            m_terrainMaterial = material;
+            UpdateTerrainMaterial();
+        }
         bool IsInitialized() const;
         void Reset();
         bool UpdateSrgIndices(const AZ::Data::Instance<AZ::RPI::ShaderResourceGroup>& srg);
