diff --git a/TerrainRenderer/TerrainFeatureProcessor.cpp b/TerrainRenderer/TerrainFeatureProcessor.cpp
index ac86f3b..9e3fe0d 100644
--- a/TerrainRenderer/TerrainFeatureProcessor.cpp
+++ b/TerrainRenderer/TerrainFeatureProcessor.cpp
@@ -77,7 +77,7 @@ namespace Terrain
     {
         AZ::Data::AssetBus::Handler::BusDisconnect();
         AzFramework::Terrain::TerrainDataNotificationBus::Handler::BusDisconnect();
-        
+
         DisableSceneNotification();
         OnTerrainDataDestroyBegin();
 
@@ -107,7 +107,7 @@ namespace Terrain
         }
         else
         {
-            m_materialInstance = AZ::RPI::Material::FindOrCreate(m_materialAsset);
+            m_materialInstance = AZ::RPI::Material::Create(m_materialAsset);
             PrepareMaterialData();
             m_terrainBoundsNeedUpdate = true;
         }
@@ -122,7 +122,7 @@ namespace Terrain
     {
         m_zBounds = {};
     }
-    
+
     void TerrainFeatureProcessor::OnTerrainDataChanged([[maybe_unused]] const AZ::Aabb& dirtyRegion, TerrainDataChangedMask dataChangedMask)
     {
         if ((dataChangedMask & TerrainDataChangedMask::Settings) == TerrainDataChangedMask::Settings)
@@ -173,7 +173,7 @@ namespace Terrain
                     // skip the rest of shader items
                     return false;
                 }
-                return true;                     
+                return true;
             });
 
         AZ_Error(TerrainFPName, m_terrainSrg, "Terrain Srg not found on any shader in the terrain material");
@@ -188,7 +188,7 @@ namespace Terrain
             {
                 m_macroMaterialManager.Initialize(m_terrainSrg);
             }
-            
+
             if (m_detailMaterialManager.IsInitialized())
             {
                 m_detailMaterialManager.UpdateSrgIndices(m_terrainSrg);
@@ -225,7 +225,7 @@ namespace Terrain
     void TerrainFeatureProcessor::ProcessSurfaces(const FeatureProcessor::RenderPacket& process)
     {
         AZ_PROFILE_FUNCTION(AzRender);
-        
+
         if (m_zBounds.m_min == 0.0f && m_zBounds.m_max == 0.0f)
         {
             return;
@@ -309,7 +309,7 @@ namespace Terrain
     {
         m_detailMaterialManager.SetDetailMaterialConfiguration(config);
     }
-    
+
     void TerrainFeatureProcessor::SetMeshConfiguration(const MeshConfiguration& config)
     {
         m_meshManager.SetConfiguration(config);
@@ -320,7 +320,7 @@ namespace Terrain
     {
         m_clipmapManager.SetConfiguration(config);
     }
-    
+
     void TerrainFeatureProcessor::CachePasses()
     {
         m_passes.clear();
