--- a/TerrainModule.cpp
+++ b/TerrainModule.cpp
@@ -19,8 +19,6 @@
 #include <Components/TerrainSurfaceGradientListComponent.h>
 #include <Components/TerrainSurfaceDataSystemComponent.h>
 #include <Components/TerrainPhysicsColliderComponent.h>
-#include <TerrainRenderer/Components/TerrainSurfaceMaterialsListComponent.h>
-#include <TerrainRenderer/Components/TerrainMacroMaterialComponent.h>
 
 namespace Terrain
 {
@@ -28,14 +26,12 @@
         : AZ::Module()
     {
         m_descriptors.insert(m_descriptors.end(), {
-                TerrainSurfaceMaterialsListComponent::CreateDescriptor(),
                 TerrainSystemComponent::CreateDescriptor(),
                 TerrainWorldComponent::CreateDescriptor(),
                 TerrainWorldDebuggerComponent::CreateDescriptor(),
                 TerrainWorldRendererComponent::CreateDescriptor(),
                 TerrainHeightGradientListComponent::CreateDescriptor(),
                 TerrainLayerSpawnerComponent::CreateDescriptor(),
-                TerrainMacroMaterialComponent::CreateDescriptor(),
                 TerrainSurfaceGradientListComponent::CreateDescriptor(),
                 TerrainSurfaceDataSystemComponent::CreateDescriptor(),
                 TerrainPhysicsColliderComponent::CreateDescriptor()
